src/domain/community/user/user.resolver.mutations.ts

```diff
@@ -214,4 +229,47 @@ export class UserResolverMutations {
     );
     return await this.userAuthorizationService.applyAuthorizationPolicy(user);
   }
+
+  private async assignUserToOrganizationByDomain(
+    agentInfo: AgentInfo,
+    user: IUser
+  ): Promise<void> {
```

Comment:
it will be better to have the Promise<boolean> as return type, as it is good to know whether the user was successfully assigned or not